src/app/SharpRaven/Data/JsonPacket.cs

```diff
@@ -123,7 +123,16 @@ private static object Merge(SentryEvent @event)
                 }
             }
             else
-                result = JObject.FromObject(extra);
+            {
+                try
+                {
+                    result = JObject.FromObject(extra);
+                }
+                catch (ArgumentException e)
+                {
+                    result = JObject.Parse(string.Format(@"{{""{0}"":""{1}""}}", extra.GetType(), extra));
```

Comment:
Yeah there's `Dictionary`... I'm too obsessed with the dynamic type thing in JSON and forgot about it. Should it be `Dictionary<string, string>`, though? Or it's handled by Json.Net itself?